public/manifest.json

```diff
@@ -2,15 +2,15 @@
   "short_name": "Juicebox",
   "name": "Juicebox",
   "description": "Community funding for people and projects.",
-  "iconPath": "android-chrome-144x144.png",
+  "iconPath": "gnosis.png",
```

Comment:
Looks like they support SVG. WDYT about finding an SVG to use instead?

Reply:
Added in 5285c2f